docs/guides/data-binding/inotifypropertychanged.md

```diff
@@ -1,21 +1,29 @@
 ---
 id: inotifypropertychanged
-title: How to use INotifyPropertyChanged
+title: How to Use INotifyPropertyChanged
 ---
 
 ## Introduction
-The `INotifyPropertyChanged` interface is a critical component in the Model-View-ViewModel (MVVM) design pattern that helps create scalable and maintainable applications. By notifying that a property has been changed, it allows the View to update automatically, improving the communication between the components of your application.
+The `INotifyPropertyChanged` interface is a critical component in the Model-View-ViewModel (MVVM) design pattern that 
+helps create scalable and maintainable applications. By notifying that a property has been changed, it allows the View 
+to update automatically, improving the communication between the components of your application.
```

Comment:
Not sure, are these line breaks necessary here?

Reply:
No. They're only for text editor purposes and aren't visually represented in the site.

I should probably be using Soft Wrap in Rider, but there's been a mix of incredibly long lines and manually formatted line breaks in docs since before I started committing. There's no style guide here.

Should I revert that part or just use soft wrap in the future?

Reply:
Probably better to revert, keeping git diff to what was actually changed.